src/compas_rhino/geometry/brep/brep.py

```diff
@@ -253,80 +335,106 @@ def from_extrusion(cls, curve, vector, cap_ends=True):
+
+        return cls.from_native(result)
 
     @classmethod
-    def from_cylinder(cls, cylinder):
-        """Create a RhinoBrep from a box.
+    def from_mesh(cls, mesh):
+        """Create a RhinoBrep from a mesh.
 
         Parameters
         ----------
-        box : :class:`compas.geometry.Box`
-            The box geometry of the brep.
+        mesh : :class:`compas.datastructures.Mesh`
+            The source mesh.
 
         Returns
         -------
         :class:`compas_rhino.geometry.RhinoBrep`
 
         """
-        rhino_cylinder = cylinder_to_rhino(cylinder)
-        return cls.from_native(rhino_cylinder.ToBrep(True, True))
+        rhino_mesh = mesh_to_rhino(mesh)
+        return cls.from_native(Rhino.Geometry.Brep.CreateFromMesh(rhino_mesh, True))
 
     @classmethod
-    def from_mesh(cls, mesh):
-        """Create a RhinoBrep from a mesh.
+    def from_native(cls, rhino_brep):
+        """Constructs a RhinoBrep from an instance of a Rhino.Geometry.Brep.
 
         Parameters
         ----------
-        mesh : :class:`compas.datastructures.Mesh`
-            The source mesh.
+        rhino_brep : :class:`Rhino.Geometry.Brep`
+            The instance of Rhino brep to wrap.
 
         Returns
         -------
         :class:`compas_rhino.geometry.RhinoBrep`
 
         """
-        rhino_mesh = mesh_to_rhino(mesh)
-        return cls.from_native(Rhino.Geometry.Brep.CreateFromMesh(rhino_mesh, True))
+        brep = cls()
+        brep._brep = rhino_brep
+        return brep
 
     @classmethod
-    def from_loft(cls, curves):
-        """Construct a Brep by lofting a set of curves.
+    def from_sphere(cls, sphere):
+        """Create a RhinoBrep from a sphere.
 
         Parameters
         ----------
-        curves : list[:class:`compas.geometry.Curve`]
+        sphere : :class:`compas.geometry.Sphere`
+            The source sphere.
 
         Returns
         -------
-        :class:`compas.geometry.Brep`
+        :class:`compas_rhino.geometry.RhinoBrep`
 
         """
-        rhino_curves = [curve_to_rhino(curve) for curve in curves]
-        start = Rhino.Geometry.Point3d.Unset
-        end = Rhino.Geometry.Point3d.Unset
-        loft_type = Rhino.Geometry.LoftType.Normal
+        rhino_sphere = sphere_to_rhino(sphere)
+        return cls.from_native(rhino_sphere.ToBrep())
 
-        results = Rhino.Geometry.Brep.CreateFromLoft(rhino_curves, start, end, loft_type, closed=False)
-        if not results:
-            raise BrepTrimmingError("Loft operation ended with no result")
-        result = results[0]
+    @classmethod
+    def from_step(cls, filepath):
+        """Construct a RhinoBrep from a STEP file.
 
-        return cls.from_native(result)
+        Parameters
+        ----------
+        filepath : str
+            The path to the step file.
+
+        Returns
+        -------
+        :class:`compas_rhino.geometry.RhinoBrep`
+
+        """
+        rs.Command('_-Import "' + filepath + '" _Enter', False)
```

Comment:
this would import any kind of file, right?
just thinking out loud, could be nice to maybe do this in the future using [Rhino.FileIO.FileStp](https://developer.rhino3d.com/api/rhinocommon/rhino.fileio.filestp)
although that one also draws directly to a `RhinoDoc`, so not very clean. a bit more explicit though.

Anyways, maybe this is good enough for now